src/yz_solrq_worker.erl

```diff
@@ -351,7 +351,7 @@ handle_cast(drain_complete, #state{indexqs = IndexQs} = State) ->
     NewIndexQs = dict:fold(
         fun(Index, #indexq{queue = Queue, queue_len = QueueLen, aux_queue = AuxQueue} = IndexQ, IndexQAccum) ->
             NewIndexQ = IndexQ#indexq{
-                queue = queue:join(Queue, AuxQueue),
+                queue = queue:join(AuxQueue, Queue),
```

Comment:
Isn't the original `queue:join(Queue, AuxQueue)` the right solution?

AuxQueue accumulates while the drain is ongoing and once that is done what will be in the Queue?

If it is leftovers from the drain, then AuxQueue should go at the end. Or could it be other things?